pkg/logutil/log.go

```diff
@@ -114,15 +114,20 @@ func InitLogger(cfg *Config) error {
 	if err != nil {
 		return errors.Trace(err)
 	}
-	err = initSaramaLogger(level)
-	if err != nil {
-		return err
-	}
+
 	err = initGRPCLogger(level)
 	if err != nil {
 		return err
 	}
 
+	if cfg.File == "" {
```

Comment:
What is the meaning here?

Reply:
We only want sarama log to be output to stdout in a cdc cli cmd. Need to be discussed.

Reply:
There is no need to output sarama log to stdout when creating a changefeed. I will add the troubleshooting description in doc.